### ISSUE TITLE
Errors and warnings through op_kernel.cc and core/grappler/utils.cc

### ISSUE THREAD

**Issue by Gandhi-Sagar** [tensorflow/tensorflow#14432]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1. It must be a bug or a feature request.
2. The form below must be filled out.
3. It shouldn't be a TensorBoard issue. Those go [here](https://github.com/tensorflow/tensorboard/issues).

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.4.0
- **Python version**: 3.6.1
- **Bazel version (if compiling from source)**: 0.7.0
- **GCC/Compiler version (if compiling from source)**: 5.4.0
- **CUDA/cuDNN version**: CUDA-8.0, cuDNN - 6.1
- **GPU model and memory**: GTX 1050, 4GB
- **Exact command to reproduce**: sample code to validate installation

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

I installed tensorflow from sources.
There was no errors during configure as well as build.  logs are below.

Further, If I run neural network example from the repository [here](https://github.com/aymericdamien/TensorFlow-Examples/blob/master/examples/3_NeuralNetworks/neural_network.py), I get the warnings
But code runs, gives output as expected. logs are below.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.
********************************** log for validation script ***************************************************
Python 3.6.1 |Anaconda custom (64-bit)| (default, May 11 2017, 13:09:58) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
>>> hello = tf.constant('Hello, Tensorflow!')
>>> sess = tf.Session()
2017-11-09 21:53:38.923408: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:900] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2017-11-09 21:53:38.924184: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1061] Found device 0 with properties: 
name: GeForce GTX 1050 major: 6 minor: 1 memoryClockRate(GHz): 1.493
pciBusID: 0000:01:00.0
totalMemory: 3.95GiB freeMemory: 3.51GiB
2017-11-09 21:53:38.924230: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1151] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1050, pci bus id: 0000:01:00.0, compute capability: 6.1)
>>> print(sess.run(hello))
2017-11-09 21:54:07.579195: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "DenseToSparseBatchDataset" device_type: "CPU"') for unknown op: DenseToSparseBatchDataset
2017-11-09 21:54:07.579255: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "GroupByWindowDataset" device_type: "CPU"') for unknown op: GroupByWindowDataset
2017-11-09 21:54:07.579261: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "IgnoreErrorsDataset" device_type: "CPU"') for unknown op: IgnoreErrorsDataset
2017-11-09 21:54:07.579286: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "DatasetToSingleElement" device_type: "CPU"') for unknown op: DatasetToSingleElement
2017-11-09 21:54:07.579301: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "DeserializeIterator" device_type: "CPU"') for unknown op: DeserializeIterator
2017-11-09 21:54:07.579307: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "MapAndBatchDataset" device_type: "CPU"') for unknown op: MapAndBatchDataset
2017-11-09 21:54:07.579324: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "ParallelInterleaveDataset" device_type: "CPU"') for unknown op: ParallelInterleaveDataset
2017-11-09 21:54:07.579336: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "ScanDataset" device_type: "CPU"') for unknown op: ScanDataset
2017-11-09 21:54:07.579345: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "SqlDataset" device_type: "CPU"') for unknown op: SqlDataset
2017-11-09 21:54:07.579410: E tensorflow/core/framework/op_kernel.cc:1142] OpKernel ('op: "SerializeIterator" device_type: "CPU"') for unknown op: SerializeIterator
b'Hello, Tensorflow!'
```
****************************************************************************************************************

********************************** log for neural network code from examples *****************************
Successfully downloaded train-images-idx3-ubyte.gz 9912422 bytes.
Extracting /tmp/data/train-images-idx3-ubyte.gz
Successfully downloaded train-labels-idx1-ubyte.gz 28881 bytes.
Extracting /tmp/data/train-labels-idx1-ubyte.gz
Successfully downloaded t10k-images-idx3-ubyte.gz 1648877 bytes.
Extracting /tmp/data/t10k-images-idx3-ubyte.gz
Successfully downloaded t10k-labels-idx1-ubyte.gz 4542 bytes.
Extracting /tmp/data/t10k-labels-idx1-ubyte.gz
WARNING:tensorflow:Using temporary folder as model directory: /tmp/tmpvak_s5_3
2017-11-09 22:31:36.069140: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:900] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2017-11-09 22:31:36.069505: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1061] Found device 0 with properties: 
name: GeForce GTX 1050 major: 6 minor: 1 memoryClockRate(GHz): 1.493
pciBusID: 0000:01:00.0
totalMemory: 3.95GiB freeMemory: 3.50GiB
2017-11-09 22:31:36.069519: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1151] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1050, pci bus id: 0000:01:00.0, compute capability: 6.1)
2017-11-09 22:31:36.368840: W tensorflow/core/grappler/utils.cc:48] Node dense_3/MatMul_fused is not in the graph.
2017-11-09 22:31:36.368900: W tensorflow/core/grappler/utils.cc:48] Node dense_2/MatMul_fused is not in the graph.
2017-11-09 22:31:36.368918: W tensorflow/core/grappler/utils.cc:48] Node dense/MatMul_fused is not in the graph.
2017-11-09 22:31:36.368996: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense_3/MatMul_grad/MatMul_1_fused is not in the graph.
2017-11-09 22:31:36.369024: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense_3/MatMul_grad/MatMul_fused is not in the graph.
2017-11-09 22:31:36.369045: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense_2/MatMul_grad/MatMul_1_fused is not in the graph.
2017-11-09 22:31:36.369074: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense_2/MatMul_grad/MatMul_fused is not in the graph.
2017-11-09 22:31:36.369097: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense/MatMul_grad/MatMul_1_fused is not in the graph.
2017-11-09 22:31:36.369122: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense/MatMul_grad/MatMul_fused is not in the graph.
2017-11-09 22:31:36.526452: W tensorflow/core/grappler/utils.cc:48] Node dense_3/MatMul_fused is not in the graph.
2017-11-09 22:31:36.526488: W tensorflow/core/grappler/utils.cc:48] Node dense_2/MatMul_fused is not in the graph.
2017-11-09 22:31:36.526504: W tensorflow/core/grappler/utils.cc:48] Node dense/MatMul_fused is not in the graph.
2017-11-09 22:31:36.526534: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense_3/MatMul_grad/MatMul_1_fused is not in the graph.
2017-11-09 22:31:36.526543: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense_3/MatMul_grad/MatMul_fused is not in the graph.
2017-11-09 22:31:36.526562: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense_2/MatMul_grad/MatMul_1_fused is not in the graph.
2017-11-09 22:31:36.526570: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense_2/MatMul_grad/MatMul_fused is not in the graph.
2017-11-09 22:31:36.526589: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense/MatMul_grad/MatMul_1_fused is not in the graph.
2017-11-09 22:31:36.526597: W tensorflow/core/grappler/utils.cc:48] Node gradients/dense/MatMul_grad/MatMul_fused is not in the graph.
2017-11-09 22:31:38.129805: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1151] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1050, pci bus id: 0000:01:00.0, compute capability: 6.1)
2017-11-09 22:31:38.168010: W tensorflow/core/grappler/utils.cc:48] Node dense_3/MatMul_fused is not in the graph.
2017-11-09 22:31:38.168065: W tensorflow/core/grappler/utils.cc:48] Node dense_2/MatMul_fused is not in the graph.
2017-11-09 22:31:38.168081: W tensorflow/core/grappler/utils.cc:48] Node dense/MatMul_fused is not in the graph.
Testing Accuracy: 0.9167

*****************************************************************************************************************

**Comment by bignamehyp** [comment]
I apologize but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please resubmit and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new) . Please provide all the information it asks. Thank you.

**Comment by hossein-h** [comment]
I have faced the same problem.
